frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetailsPage.tsx

```diff
@@ -140,7 +178,7 @@ export function PolicyTemplateDetailsPage() {
         />
       }
     >
-      <Suspense fallback={<Fragment />}>
+      <Suspense fallback={<></>}>
```

Comment:
What's the difference between this and `<Fragment>`?

Reply:
same

Reply:
i got a sonar warning? something so I changed